src/m365/spo/commands/listitem/listitem-add.spec.ts

```diff
@@ -71,7 +71,7 @@ describe(commands.LISTITEM_ADD, () => {
           "Created": "2018-03-15T10:43:10Z",
           "EditorId": 3,
           "GUID": "ea093c7b-8ae6-4400-8b75-e2d01154dffc",
-          "ID": actualId,
+          "Id": actualId,
```

Comment:
We should have both Id and ID in this response to make the API response as realistic as possible.